### PR TITLE
create base html structure for index.html

### PR DESCRIPTION
**Pull request by Xiphe**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,2 +1,12 @@
...
+<!doctype html>
+<html lang="de">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Nancy Heiber</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+    <h1>Hallo Welt</h1>
```
Comment (giowaza) [inline index.html]: Please add Nancy Heiber Dressage.

Comment (Xiphe) [reply index.html]: Sure, will do that later on

Comment (giowaza) [reply index.html]: Alright!

Comment (giowaza) [reply index.html]: No worries. I'll take care of it!

**Pull request merged**